docs/nidcpower/functions.rst

```diff
@@ -1035,15 +1035,15 @@ nidcpower.Session methods
     The compliance limit is the current limit when the output function is
     set to NIDCPOWER\_VAL\_DC\_VOLTAGE. If the output is operating at the
     compliance limit, the output reaches the current limit before the
-    desired voltage level. Refer to the :py:func:`nidcpower.configure_output_function`
-    function and the :py:func:`nidcpower.configure_current_limit` function for more
+    desired voltage level. Refer to the :py:func:`nidcpower.ConfigureOutputFunction`
+    function and the :py:func:`nidcpower.ConfigureCurrentLimit` function for more
```

Comment:
nidcpower.ConfigureCurrentLimit makes no sense, it's not even the C function name,